src/native/managed/cdac/mscordaccore_universal/Entrypoints.cs

```diff
@@ -147,6 +148,7 @@ private static unsafe int CLRDataCreateInstanceImpl(Guid* pIID, IntPtr /*ICLRDat
                     return dataTarget.GetThreadContext(threadId, contextFlags, (uint)bufferToFill.Length, bufferPtr);
                 }
             },
+            null,
```

Comment:
I'd prefer having an empty collection being passed instead of `null`.
```suggestion
            [],
```